minerals/startvesktop

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Set FEATURES to what goes after --enable-features; for example, UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc will then get turned into --enable-features=UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc.
+FEATURES="UseSkiaRenderer"
+TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-dev.vencord.Vesktop}"
+declare -a FLAGS=(--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --disable-gpu-blocklist)
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]]
+then
+    FEATURES="${FEATURES},WebRTCPipeWireCapturer"
+
+    WAYLAND_SOCKET=${WAYLAND_DISPLAY:-"wayland-0"}
+
+    echo "Using Wayland, enabling PipeWire for video capture."
+    echo "Screenshare may be broken on first start."
+    echo "Make sure you have the legacy screencapturer enabled."
+
+    FEATURES="${FEATURES},WebRTCPipeWireCapturer"
```

Comment:
this line seems to be duplicated. is it even necessary? i'm fairly certain it's obsolete and our electron version - at least i never passed it and it works anyway